src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationPreambleController.cs

```diff
@@ -346,6 +358,9 @@ public async Task<IActionResult> SelectApplicationRoute()
             {
                 model.ApplicationRouteId = applicationDetails.ApplicationRoute.Id;
             }
+            var getHelpSessionKey = string.Format(GetHelpSubmittedForPageKey, "ApplicationRoute");
```

Comment:
As mentioned above GetHelpSubmittedForPageKey is a format string and not a key. Please fix

Reply:
Renamed constant name